device/bgp/bgpsrteadvanced.yaml

```diff
@@ -0,0 +1,25 @@
+components:
+  schemas:
+    BgpSrTe.Advanced:
+      description: >-
+        Configuration for advanced BGP route range settings.
+      type: object
+      properties:
+        origin:
+          description: >-
+            The origin value. 
+            A null value means the origin feature is not enabled.
+          type: string
+          enum:
+          - igp
+          - egp
+        multi_exit_discriminator:
+          description: >-
+            The multi exit discriminator (MED) value. 
+            A null value means the MED feature is not enabled.
+          type: integer
+        local_preference:
+          description: >-
+            Local Preference attribute, used by iBGP peers to select 
```

Comment:
Local Preference attribute, for iBGP peers to select the path for the traffic leaving the AS.